core/client/fs/src/main/java/alluxio/client/block/StreamFactory.java

```diff
@@ -42,30 +47,17 @@ private StreamFactory() {} // prevent instantiation
    * @param options the out stream options
    * @return the {@link OutputStream} object
    */
-  public static BlockOutStream createLocalBlockOutStream(FileSystemContext context, long blockId,
+  public static BlockOutStream createBlockOutStream(FileSystemContext context, long blockId,
```

Comment:
Do we still need StreamFactory now? it becomes very simple with only two methods, and I think the logic can be consolidated to `BlockInStream` and `BlockOutStream` rather than having a separate static factory class here.